.github/workflows/lint.yaml

```diff
@@ -53,11 +53,15 @@ jobs:
           fetch-depth: 0
       - name: Run Checkton
         id: checkton
-        uses: chmeliik/checkton@v0.1.2 
+        uses: chmeliik/checkton@v0.2.2 
         # Migrating to the konflux-ci org
         with:
           fail-on-findings: true
-          find-copies-harder: true
+          find-copies-harder: false
+          # Set to false to disable the differential check
+          differential: false
+          # Add a regex to include only the files you want to check
+          include-regex: "tasks/collect-data/.*\\.yaml"
```

Comment:
Opening a thread here so it's not accidentally merged with it.

Reply:
Restored